basic/src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from "react-router-dom";
+import "./App.css";
+import { Header } from "./common/Header";
+
+export function App() {
+  return (
+    <main className="flex min-h-screen w-full flex-col bg-neutral-300 text-black">
```

Comment:
```suggestion
    <main className="flex min-h-screen w-full flex-col bg-neutral-50 text-black">
```
Just a personal preference, I used the same shade in the Kitchen Sink App https://github.com/wasp-lang/wasp/pull/2812/files#diff-b518782a0ea8e0b7a1a883039b7352a33d9ddd94fff2bdf5f43d22dcc1ae8fa4R24

Reply:
Doesn't really work with the current style (`bg-white` cards).
I will leave it for when I re-do the styling.
<img width="651" alt="image" src="https://github.com/user-attachments/assets/127aac26-764c-4d8f-b716-ba48b562e9fe" />